actuator.c

```diff
@@ -1,56 +1,50 @@
-#include <xc.h>
-#include <stdint.h>
 #include <stdbool.h>
+#include <stdint.h>
+#include <xc.h>
 
-#include "canlib/can.h"
-#include "canlib/can_common.h"
-#include "canlib/pic18f26k83/pic18f26k83_can.h"
-#include "canlib/message_types.h"
-#include "canlib/util/can_tx_buffer.h"
-
-#include "mcc_generated_files/mcc.h"
-
+#include "IOExpanderDriver.h"
 #include "actuator.h"
-#include "board.h"
-#include "timer.h"
+#include "canlib/canlib.h"
+#include "sensor_general.h"
 
-void actuator_init(){
-    TRISB5 = 0; // set ACTUATOR_CONTROL (pin 26) as output
-    if (SAFE_STATE == ACTUATOR_OFF) {
-        RELAY_OFF();
-    } else if (SAFE_STATE == ACTUATOR_ON) {
-        RELAY_ON();
-    } else {
-        // Should not get here.
-        while (1) {}; // panic, this will be caught early on in testing
-    }
-    
-#ifdef INJECTOR
-    LATB4 = 1;
-    TRISB4 = 0; // set LIMIT_OPEN as output to power hall sensor
-
-    TRISB3 = 1; // set LIMIT_CLOSED (pin 24) as input
-    ANSELB3 = 1; // set as analog input
-#else
-    TRISB4 = 1; // set LIMIT_OPEN (pin 25) as input
-    ANSELB4 = 0; // set as digital input
+uint8_t actuator_states = 0;
```

Comment:
We never actually read actuator states, so we should probs remove this variable and just have the set use the pca, no need to maintain the state of the actuator in the code